### PR TITLE
feature: update dates, call (length/award/format link), pc list and twitter link

### PR DESCRIPTION
**Pull request by natedingyifeng**
1. Update submission and notification dates
2. Update the length of submissions
3. Update award information
4. Update the PC list
5. Update format link
6. Add twitter link in the header

@UniverseFly @ganler Plz check these changes, thanks!

### CODE REVIEW

File: _includes/description.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <!-- <img src="/assets/llm4code-website-light.svg" width="100%"> -->
 <ul class="custom-ul">
   <!-- <li><img src="/assets/llm4code-website-light.svg" width="100%"></li> -->
-  <li> 🤖 The First International Workshop on Large Language Models for Code</li>
+  <li> 🤖 The First International Workshop on Large Language Models for Code (Follow us on Twitter: <a href="https://twitter.com/llm4code">@llm4code</a>)</li>
```
Comment (ganler) [inline _includes/description.md]: How about adding another specific line with a Twitter logo?

File: call.md [hunk 2]
```diff
@@ -11,10 +11,12 @@ permalink: /call/
 
 This workshop accepts both research papers and position papers:
 
-- **Research papers** (4 -- 8 pages) on novel approaches, tools, datasets, or studies.
-- **Position papers** (1 -- 4 pages) on novel ideas and positions that have yet to be fully developed.
+- **Research papers (4 -- 8 pages inc. references)** on novel approaches, tools, datasets, or studies.
+- **Position papers (1 -- 4 pages inc. references)** on novel ideas and positions that have yet to be fully developed.
```
Comment (ganler) [inline call.md]: just say "including"

**Review by ganler — CHANGES_REQUESTED** [review]

File: call.md (hunk 2 above)
Comment (natedingyifeng) [reply call.md]: Done.

File: _includes/description.md (hunk 1 above)
Comment (natedingyifeng) [reply _includes/description.md]: Added

File: call.md [hunk 3]
```diff
@@ -11,10 +11,12 @@ permalink: /call/
...
-- **Position papers** (1 -- 4 pages) on novel ideas and positions that have yet to be fully developed.
+- **Research papers (4 -- 8 pages including references)** on novel approaches, tools, datasets, or studies.
+- **Position papers (1 -- 4 pages including references)** on novel ideas and positions that have yet to be fully developed.
 
-All papers will be submitted via HotCRP and be reviewed in double-blinded. The submission should comply with the ACM format (in line with ICSE 2024) and should present the original contribution. At least one author of each accepted paper should register for the workshop and present the paper in the workshop.
+All papers will be submitted via HotCRP and be reviewed in double-blinded. The submission should comply with the ACM format (in line with [ICSE 2024](https://conf.researchr.org/track/icse-2024/icse-2024-research-track#submission-process)) and should present the original contribution. At least one author of each accepted paper should register for the workshop and present the paper in the workshop.
+
+**:trophy: We will announce the best paper and the best presentation awards during the workshop!**
```
Comment (UniverseFly) [inline call.md]: `:trophy:` is not displaying as an emoji from my local build. try 🏆

File: _includes/description.md (hunk 1 above)
Comment (UniverseFly) [reply _includes/description.md]: @natedingyifeng You can download $\mathbb X$ here https://about.twitter.com/en/who-we-are/brand-toolkit

**Review by UniverseFly — CHANGES_REQUESTED** [review]

File: call.md (hunk 3 above)
Comment (natedingyifeng) [reply call.md]: Done.

**Review by UniverseFly — APPROVED** [review]
LGTM

**Review by UniverseFly — APPROVED** [review]

**Review by ganler — APPROVED** [review]
LGTM

**Pull request merged**